### ISSUE TITLE
Update the AppDesigner and PropertyPages to be properly themed.

### ISSUE THREAD

**Issue by tannergooding** [dotnet/project-system#2849]
As part of the general accessibility work, the AppDesigner and PropertyPages have been updated to properly support theming (https://github.com/dotnet/project-system/pull/2158).

However, the behavior is currently "opt-in" and the property pages we provide are not themed:
![image](https://user-images.githubusercontent.com/10487869/30821596-bb141da8-a1da-11e7-8406-8be46b100e87.png)

For our property pages to consume the new theming behavior, the following changes need to be made:
1.	For any classes that inherit from `Microsoft.VisualStudio.Editors.PropertyPages.PropPageBase`, you need to override the `SupportsTheming` property and return `true` (https://github.com/dotnet/project-system/pull/2158/files#diff-6b6cdb996a525002b3ac123f55594dc0R274)
2.	For any classes that inherit from `Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase`, you need to override the `SupportsTheming` property and return `true` (https://github.com/dotnet/project-system/pull/2158/files#diff-81707917a7981f2eb7606e2abfd61343R322)
3.	Additionally, update your property pages in general to listen to the `IVsBroadcastMessageEvents.OnBroadcastMessage` events (the WM_PALETTECHANGE, WM_SYSCOLORCHANGE, and WM_THEMECHANGED messages) and configure them to pull their colors from the VS Theming service (https://github.com/dotnet/project-system/pull/2158/files#diff-81707917a7981f2eb7606e2abfd61343R3854)

**Comment by natidea** [comment]
Related VSFeedback Issue: [VSFeedback](https://developercommunity.visualstudio.com/content/problem/116849/c-project-property-pages-are-incorrectly-themed-in.html) | [497645](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/497645)

**Comment by paulvanbrenk** [comment]
Is this still happening for 15.7, we have very little runway to update the Node Tools and TypeScript at this point.

**Comment by jmarolf** [comment]
@paulvanbrenk I do not think this is going to happen for 15.7 at this point. @Pilchie to confirm

**Comment by Pilchie** [comment]
Agreed - this is not the sort of thing we will do in a scoped release like 15.7.

**Comment by drewnoakes** [comment]
The new Project Properties UI supports theming. We will not be investing in theme support for the legacy UI at this point.
